bears/general/FilenameBear.py

```diff
@@ -43,6 +45,10 @@ def run(self, filename, file,
         :param ignore_uppercase_filenames:
             Whether or not to ignore fully uppercase filenames completely,
             e.g. COPYING, LICENSE etc.
+        :param prefix:
+            Check whether the filename uses a certain prefix.
+        :param suffix:
+            Check whether the filename uses a certain suffix.
```

Comment:
I thought that this is obvious especially for the suffix part.
Anyway, i will update the description.